template/.polarion/tracker/fields/workitem-link-role-enum.xml

```diff
@@ -2,242 +2,159 @@
+        <rule from="operationalEntity" to="operationalActivity" />
+    </option>
+    <option id="inputs" description="An OperationalActivity has incoming OperationalInteractions which transport input data/exchange items." name="incoming" oppositeName="is input on" sortOrder="111">
+        <rule from="operationalActivity" to="operationalInteraction" />
+    </option>
+    <option id="outputs" description="An OperationalActivity has outgoing OperationalInteractions which transport output data/exchange items." name="outgoing" oppositeName="is output on" sortOrder="112">
+        <rule from="operationalActivity" to="operationalInteraction" />
+    </option>
+    <option id="exchanges" description="The exchanges connecting the Function." name="exchange" oppositeName="is exchange on" sortOrder="110">
+        <rule from="systemFunction, logicalFunction, physicalFunction" to="functionalExchange" />
+    </option>
+    <option id="input_exchanges" description="Incoming FunctionalExchange on any Function outside of the Operational layer." name="incoming" oppositeName="is input exchange on" sortOrder="113">
+        <rule from="systemFunction, logicalFunction" to="functionalExchange" />
+    </option>
+    <option id="output_exchanges" description="Outgoing FunctionalExchange on any Function outside of the Operational layer." name="outgoing" oppositeName="is output exchange on" sortOrder="114">
+        <rule from="systemFunction, logicalFunction" to="functionalExchange" />
+    </option>
+    <option id="allocated_functions" description="Allocated Functions on a Component." name="allocates" oppositeName="is allocated by" sortOrder="115">
+        <rule from="systemComponent, logicalComponent, logicalActor" to="systemFunction, logicalFunction" />
+    </option>
+    <option id="constraints" description="Constraints on an object." name="constraint by" oppositeName="constraints">
+        <rule from="operationalCapability, systemCapability, capabilityRealization" to="constraint" />
+    </option>
+    <option id="scenarios" description="Scenarios under this Capability." name="scenario" oppositeName="scenario on" sortOrder="112">
+        <rule from="capability, capabilityRealization" to="scenario" />
+    </option>
+    <option id="diagram_elements" description="Linking to elements viewable in a diagram." name="shows" oppositeName="is shown in">
+        <rule from="diagram" />
+    </option>
+    <option id="realized_functions" description="A Function realizes a lower-layer Function." name="realized" oppositeName="realized in">
+        <rule from="physicalFunction" to="logicalFunction" />
+        <rule from="logicalFunction" to="systemFunction" />
+        <rule from="systemFunction" to="operationalActivity" />
+    </option>
+    <option id="physical_links" name="physical links" oppositeName="physical link of" sortOrder="118">
+        <rule from="componentExchange, physicalComponentNode, physicalActor" to="physicalLink"/>
```

Comment:
I think we miss the `exchange` role here, aren't we?

Reply:
There is `exchanges` with the name exchange.